pmd-java/src/main/java/net/sourceforge/pmd/lang/java/ast/FormalComment.java

```diff
@@ -4,15 +4,40 @@
 
 package net.sourceforge.pmd.lang.java.ast;
 
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Map;
+
+import net.sourceforge.pmd.lang.ast.Node;
+import net.sourceforge.pmd.lang.java.javadoc.JavadocTag;
+
 public class FormalComment extends Comment {
 
     public FormalComment(Token t) {
         super(t);
-    }
 
+        findJavadocs(t.image);
+    }
 
     @Override
     public String getXPathNodeName() {
         return "FormalComment";
     }
+
+    private void findJavadocs(String commentText) {
+        Collection<JavadocElement> kids = new ArrayList<>();
+
+        Map<String, Integer> tags = CommentUtil.javadocTagsIn(commentText);
```

Comment:
shouldn't we avoid internal usage of `CommentUtil` as we are deprecating it?

Reply:
I've missed that...